js_modules/dagster-ui/packages/ui-core/src/assets/AssetLink.tsx

```diff
@@ -24,7 +24,7 @@ export const AssetLink: React.FC<{
       style={{maxWidth: '100%'}}
     >
       {icon ? (
-        <Box margin={{right: 8, top: 2}}>
+        <Box margin={{right: 8, top: 2}} style={{width: 16}}>
```

Comment:
"16px"

Reply:
Ahh I think I can rm this, I was thinking it was causing the text truncation issue I added the note about, but turns out it was not 🙏